$NetBSD: patch-Cargo.toml,v 1.3 2023/05/01 15:11:23 pin Exp $

Don't patch crate, only needed for cross-compilling.
Only needed for Windows.

--- Cargo.toml.orig	2023-05-01 00:30:32.000000000 +0000
+++ Cargo.toml
@@ -86,7 +86,3 @@ strip = true
 
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
-
-[patch.crates-io]
-# ffmpeg-sys-next does not support cross-compilation, which I use to produce binaries https://github.com/zmwangx/rust-ffmpeg-sys/pull/30
-ffmpeg-sys-next = { rev = "94d5496d88900bdc0cad66733138134d0ea3cf31", git = "https://github.com/kornelski/rust-ffmpeg-sys-1"}
